### DISCUSSION TITLE
setting up the inputs-FASTQ.csv file

### DISCUSSION THREAD

**Discussion by maneeshathaker** [uclahs-cds/metapipeline-DNA#51] [Q&A] [answered]
Hi everyone, 

I have a dataset of four patients with matched tumor-normal samples. Of the four patients, two have multiple tumor samples. I have two questions about the set up of the input csv file. 

The simplified structure of my input csv is as follows: "Patient 1: N1, T1, T2, T3; Patient 2: N1, T1, T2, T3 ..." but I'm not sure if this is correct. 
1. I wanted to clarify if I need a separate input csv file per patient? Currently, my input file has all the patients listed in order in the same file. 
2. Do I need a separate input csv file for each tumor-normal pairing? I.e., "Patient 1: N1, T1" "Patient 1: N1, T2", so on so forth. 

@yashpatel6

**Comment by yashpatel6** [comment, marked as answer]
>     1. I wanted to clarify if I need a separate input csv file per patient? Currently, my input file has all the patients listed in order in the same file.

The metapipeline only requires one input CSV containing all of the patients, so no need to split per patient.

>     2. Do I need a separate input csv file for each tumor-normal pairing? I.e., "Patient 1: N1, T1" "Patient 1: N1, T2", so on so forth.

The metapipeline will also handle creating the pairing based on sample IDs as long as the `state` column in the input CSV is properly set for the samples (ie. for each patient, the normal sample is marked as `normal` and the tumor samples are marked as `tumor`).

**Reply by maneeshathaker** [reply]
Perfect! Thank you! :)
